apps/edxapp/templates/cms/secret_tx.yml.j2

```diff
@@ -0,0 +1,12 @@
+{% if TX_TOKEN is defined %}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: edxapp
+    service: "cms-lms"
+  name: "{{ edxapp_tx_secret_name }}"
+  namespace: "{{ project_name }}"
+data:
+  TX_TOKEN: "{{ TX_TOKEN | default('') | b64encode }}"
```

Comment:
s/TX_TOKEN/TRANSIFEX_TOKEN?  

The telecom engineer in me thinks Tx means "transmission" :-/

Reply:
`TX_TOKEN` is the expected environment variable name: https://docs.transifex.com/client/init#before-running-tx-init

I wanted to stick to that. WDYT?

Reply:
ok then! Maybe in the filename use transifex so it's at least explicit globally?